config/base/kustomization.yaml

```diff
@@ -17,16 +17,13 @@ resources:
 - 100-namespace.yaml
 - 300-operator_v1alpha1_pipeline_crd.yaml
 - 300-operator_v1alpha1_trigger_crd.yaml
-- 300-operator_v1alpha1_dashboard_crd.yaml
-- 300-operator_v1alpha1_addon_crd.yaml
 - 300-operator_v1alpha1_config_crd.yaml
-- 300-operator_v1alpha1_result_crd.yaml
```

Comment:
Why are those removed from here ?

Reply:
As dashboard, results and addons are not common in all platforms, we need to move it to respective `<overlay-name>/kustomization.yaml` instead of keeping them in `base/kustomization.yaml`.

Reply:
Ok, then the question is "why are they not common ?" 😝 `Results` for example, or others.

Reply:
ie, TektonResults are not supported in OpenShift builds yet. :slightly_smiling_face:
so when we add the TektonResult controller to OpenSHift main, we shall make this common :thumbsup:

Reply:
i have created an issue https://github.com/tektoncd/operator/issues/400
let us continue this discussion therer 🙂